wgpu/examples/water/main.rs

```diff
@@ -528,7 +535,11 @@ impl framework::Example for Example {
             // Fragment shader and output targets
             fragment: Some(wgpu::FragmentState {
                 module: &water_module,
-                entry_point: "fs_main",
+                entry_point: if read_only_depth_stencil {
```

Comment:
I realize now this won't be enough, we should have RODS validation against the downlevel limit, and that validation will still cause this to fail even if we're using the new shader - we need to completely remove the bindings from the bgl